static/app/views/aiMonitoring/PipelinesTable.tsx

```diff
@@ -237,9 +240,7 @@ function renderBodyCell(
 
     return (
       <Link
-        to={normalizeUrl(
-          `/organizations/${organization.slug}/ai-monitoring/pipeline-type/${row['span.group']}?${qs.stringify(queryString)}`
-        )}
+        to={`${moduleURL}/pipeline-type/${row['span.group']}?${qs.stringify(queryString)}`}
```

Comment:
🤔 I wonder if we can expand `useModuleUrl` even more to return us the sub paths too, because now we have the same problem with renaming with `pipeline-type` (and anything after).

Maybe 
```
{getModuleUrl} = useModuleUrl();
const url = getModuleUrl(ModuleName.AI, 'pipeline-type')
```
The second param could be optional

Reply:
Yeah, I thought about that! Definitely possible. It'll look a little awkward because these helpers are hooks, so they can't be used conditionally, and the return value has to be passed through helper functions. For just module URLs it's not _too_ bad, but even that is awkward. I wonder if later versions of React Router have built-it helpers for this, that'd be nice